docker/jenkins/periodic_db_update.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Needs DEIS_CONTROLLER URL, DEIS_USERNAME, DEIS_PASSWORD,
+# DEIS_PROFILE, and DEIS_APPLICATION environment variables.
+#
+# To set them go to Job -> Configure -> Build Environment -> Inject
+# passwords and Inject env variables
+#
+
```

Comment:
Add `set -xe`  to
 - (-x) Print commands just before execution - with all expansions and substitutions done, and words marked - useful for debugging.
 - (-e) and to exit immediately if a command fails.